src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
I don't think you have addressed this comment.

Reply:
Author response to review comment.